content/js/record.md

```diff
@@ -265,7 +268,10 @@ skygear.publicDB.save(note).then((note) => {
 - Our JIT schema at development will migrate the DB schema to sequence. All
   `noteID` at `Note` will be a sequence type once migrated.
 
-If you wish to override sequence manually, you can do that as well:
+If you wish to override sequence manually, you can do that as well. If the
+provided id is taken by another record, there will be an error; otherwise,
```

Comment:
using "id" in this paragraph will be confusing to the user. Perhaps stick with `noteID`?